src/HDInsight/HDInsightOnAks.Autorest/custom/New-AzHdInsightOnAksClusterConfigFile.ps1

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# ----------------------------------------------------------------------------------
+
+
+function New-AzHdInsightOnAksClusterConfigFile {
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.HdInsightOnAks.Models.Api20230601Preview.IClusterConfigFile])]
```

Comment:
New-AzHdInsightOnAksClusterConfigFile -> New-AzHdInsightOnAksClusterConfigFileObject as we are creating an in-memory object. Please check other cmdlets as well.